### PR TITLE
Update content and design

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
Updated the content based on the copy doc and matched the design

## QA
- Pull down and run `yarn build`
- Then open index.html
- Check that the copy matches the [copy doc](https://docs.google.com/document/d/1fmPEv9gi4kZZDDylGtUJg5KNwK7GktdMTYCF570WyYA/edit#heading=h.s107xu2kx7vo)
- Check the styling matches the [latest design](https://github.com/canonical-websites/www.ubuntu.com/issues/4049#issuecomment-420999691)

## Screenshot
screenshot_2018-09-14 canonical global navigation demo

Fixes: https://github.com/canonical-websites/www.ubuntu.com/issues/4049

### CODE REVIEW

**Comment by frankiegranato** [comment]
Still no demo but I checked the Diff and the screenshot this morming so I am adding my +1 anyway.

**Comment by anthonydillon** [comment]
There are no demos for this project and it wouldn't be trivial to add but it would be good.

**Comment by anthonydillon** [comment]
Created an issue to get demos working: https://github.com/canonical-webteam/global-nav/issues/48

**Comment by anasereijo** [comment]
@anthonydillon Products dropdown is looking great 👍 
can you please make the same changes on Login? (please change the canonical logo to the white version too)
thanks

**Comment by anthonydillon** [comment]
@Caleb-Ellis @anasereijo fixed.

**Comment by anasereijo** [comment]
@anthonydillon Login and Sign-up links should be in the same line and the paddings should stay the same. Thank you

File: src/sass/global-nav.scss [hunk 1]
```diff
@@ -309,22 +309,23 @@ $global-nav-overlay-color: rgba(17, 17, 17, .4);
...
-    margin: -1rem 0 .1rem 0;
+    margin: -1rem 0 .1rem 2.75rem;
 
     @media (max-width: $global-nav-breakpoint) {
       display: none;
     }
   }
 
   .global-nav__link {
+    display: flex;
```
Comment (Caleb-Ellis) [inline src/sass/global-nav.scss]: Changing this to inline-flex should fix the styling issues

File: src/sass/global-nav.scss [hunk 2]
```diff
@@ -350,7 +351,7 @@ $global-nav-overlay-color: rgba(17, 17, 17, .4);
 
   .global-nav__inline-list {
     margin: .5rem 0 0 0;
-    padding: 0;
+    padding: 0 0 0 2.75rem;
```
Comment (Caleb-Ellis) [inline src/sass/global-nav.scss]: and this should be 0 for smalls screens

**Comment by anthonydillon** [comment]
@caldav @anasereijo fixed your review comments

**Comment by anasereijo** [comment]
@anthonydillon can you please attach images of the Products and Login tabs? Thanks

**Comment by shivjoshi1996** [comment]
@anasereijo 

globalnav1

globalnav2

**Comment by anasereijo** [comment]
thanks for the jpgs @shivjoshi1996 👍 

@anthonydillon can we please have back the bottom padding on both dropdowns?
the text is too close to the edge of the dropdown. Please check what we have at the moment.

thanks!

**Comment by anthonydillon** [comment]
@anasereijo thanks, done.

screenshot_2018-09-20 canonical global navigation demo 1
screenshot_2018-09-20 canonical global navigation demo

**Comment by anasereijo** [comment]
awesome @anthonydillon, thanks

**Comment by anasereijo** [comment]
@deadlight design is +1 can you have a look please?

**Pull request merged**